src/hotspot/share/logging/logDecorators.cpp

```diff
@@ -24,6 +24,11 @@
 #include "precompiled.hpp"
 #include "logging/logDecorators.hpp"
 #include "runtime/os.hpp"
+#include "logDecorators.hpp"
+
+const LogLevelType AnyLevel = LogLevelType::NotMentioned;
+#define DEFAULT_DECORATORS \
```

Comment:
I think this should also have the default decorators that UL already have. That is, all data about default decorators is gathered here.